Assets/MixedRealityToolkit-SDK/Features/Input/Scripts/BaseInputHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Input
+{
+    public class BaseInputHandler : InputSystemGlobalListener
+    {
+        [SerializeField]
+        [Tooltip("Is Focus required to receive input events on this GameObject?")]
+        private bool isFocusRequired = false;
+
+        /// <summary>
+        /// Is Focus required to receive input events on this GameObject?
+        /// </summary>
+        public bool IsFocusRequired
+        {
+            get { return isFocusRequired; }
+            set { isFocusRequired = value; }
+        }
+
+        #region Monobehaviour Implementation
+
+        protected override void OnEnable()
+        {
+            if (!isFocusRequired)
```

Comment:
I originally got confused as to why nothing is done when focus IS required (until I read the doc comments for InputSystemGlobalListener). Perhaps a quick comment here would help clarify for others.

Reply:
Users should be referring to our training docs for using this handler, Code documentation is only a fall back for advanced users.  I don't think it's needed in this specific case